src/test/java/org/candlepin/util/ContentOverrideValidatorTest.java

```diff
@@ -317,25 +318,21 @@ public void testValidateWithEmptyOverrideValue() {
 
     @Test
     public void testValidateWithLongOverrideValue() {
-        List<ContentOverrideDTO> overrides = this.buildOverridesList(3);
-
-        // We expect this invocation to pass
-        this.validator.validate(overrides);
-
-        // Add our invalid override...
-        StringBuilder builder = new StringBuilder();
-        while (builder.length() < ContentOverrideValidator.MAX_VALUE_LENGTH) {
-            builder.append("longstring");
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
It's not strictly necessary to rename this variable to be so verbose. In this context we don't have any other builders to confuse it with, the declaration and lifecycle are limited to the immediate scope, and we're staying below our method length requirement, so finding the declaration from any of its uses should be trivial.

If this is purely a stylistic choice, then feel free to ignore this comment.